### PR TITLE
UI Improvements [#144405755]

### PR DESCRIPTION
**Pull request by kswenson**
UI Improvements [#144405755]
- Move bottom controls to new `ControlPanel` component
- Move sensor buttons and controls to side panel
- Show X axis label
- Enable auto-rescale button
- Singleton graph is same height as twin graphs (overall height is constant)
- Use reload icon for reload button

### CODE REVIEW

File: src/components/app.tsx [hunk 1]
```diff
@@ -412,18 +384,33 @@ export class App extends React.Component<AppProps, AppState> {
...
+                    <div>{this.state.statusMessage || "\xA0"}</div>
                 </div>
-                <div>{this.state.statusMessage}</div>
-                {this.renderGraph(this.sensor1, "graph1")}
-                {this.state.secondGraph ? this.renderGraph(this.sensor2, "graph2"): null}
-                {this.renderControls()}
+                {this.renderGraph(this.sensor1, "graph1", !this.state.secondGraph)}
+                {this.state.secondGraph
+                    ? this.renderGraph(this.sensor2, "graph2", false, true)
+                    : null}
```
Comment (pjanik) [inline src/components/app.tsx]: I usually write:
```javascript
{ 
  this.state.secondGraph &&
  this.renderGraph(this.sensor2, "graph2", false, true) 
}
```
to save one line ;)

File: src/components/sensor-graph.tsx [hunk 2]
```diff
@@ -61,8 +66,8 @@ export class SensorGraph extends React.Component<SensorGraphProps, SensorGraphSt
         this.props.sensorConnector.off("data", this.onSensorData);
     }
     
-    onUnitSelect(event:React.FormEvent<HTMLSelectElement>) {
-        this.setUnit(event.currentTarget.value);
+    onUnitSelect = (unit:string) => {
```
Comment (pjanik) [inline src/components/sensor-graph.tsx]: Nice, I didn't know this trick to auto-bind a method. ;)

**Comment by pjanik** [comment]
Looks good. I haven't really started working on this project, so not sure if it makes sense for me to review it at this point. But I skimmed through the changes and it looks good.

**Comment by kswenson** [comment]
@pjanik I realize that you haven't worked with this codebase, but you're the closest thing I've got to a reviewer and even general comments on style/techniques and possibly catching the occasional typo can still be worthwhile, so expect a few more review requests while you're still on the project. 😉

**Pull request merged**